views/js/runner/provider/qti.js

```diff
@@ -158,7 +159,14 @@ define([
                          */
                         self.trigger('listpic', picManager.collection(self._item));
 
-                        done();
+
+                        userModules.load()
```

Comment:
In that context, you can chain the promises, so you don't need to repeat the `catch` clause.

```js
return userModules.load().then(done);
```